### ISSUE TITLE
m1 SLEAP does not export training package with labeled frames

### ISSUE THREAD

**Issue by emguthman** [talmolab/sleap#905]
# Bug description

m1 SLEAP does not export training package with labeled frames

## Expected behaviour

I would get a *.pkg.slp file with labeled frames

## Actual behaviour

SLEAP does nothing and returns an error

## Your personal set up

- OS: macOS-12.3-arm64-arm-64bit

- Version(s):
SLEAP: 1.2.6
TensorFlow: 2.9.2
Numpy: 1.22.3
Python: 3.9.13

- SLEAP installation method (listed [here](https://sleap.ai/installation.html#)):
  - [ ] [Conda from package](https://sleap.ai/installation.html#conda-package)
  - [ ] [Conda from source](https://sleap.ai/installation.html#conda-from-source)
  - [ ] [pip package](https://sleap.ai/installation.html#pip-package)
  - [X] [M1 Macs](https://sleap.ai/installation.html#m1-macs)

<details><summary>Environment packages</summary>

```
# paste output of `pip freeze` or `conda list` here
```
# Name                    Version                   Build  Channel
brotlipy                  0.7.0           py39h1a28f6b_1002
ca-certificates           2022.4.26            hca03da5_0
certifi                   2022.5.18.1      py39hca03da5_0
cffi                      1.15.0           py39h22df2f2_1
charset-normalizer        2.0.4              pyhd3eb1b0_0
conda                     4.12.0           py39hca03da5_0
conda-content-trust       0.1.1              pyhd3eb1b0_0
conda-package-handling    1.8.1            py39h1a28f6b_0
cryptography              37.0.1           py39h834c97f_0
idna                      3.3                pyhd3eb1b0_0
libcxx                    12.0.0               hf6beb65_1
libffi                    3.4.2                hc377ac9_2
ncurses                   6.3                  h1a28f6b_2
openssl                   1.1.1o               h1a28f6b_0
pip                       21.2.4           py39hca03da5_0
pycosat                   0.6.3            py39h1a28f6b_0
pycparser                 2.21               pyhd3eb1b0_0
pyopenssl                 22.0.0             pyhd3eb1b0_0
pysocks                   1.7.1            py39hca03da5_0
python                    3.9.12               hbdb9e5c_0
python.app                3                py39h1a28f6b_0
readline                  8.1.2                h1a28f6b_1
requests                  2.27.1             pyhd3eb1b0_0
ruamel_yaml               0.15.100         py39h1a28f6b_0
setuptools                61.2.0           py39hca03da5_0
six                       1.16.0             pyhd3eb1b0_1
sqlite                    3.38.3               h1058600_0
tk                        8.6.11               hb8d0fd4_1
tqdm                      4.64.0           py39hca03da5_0
tzdata                    2022a                hda174b7_0
urllib3                   1.26.9           py39hca03da5_0
wheel                     0.37.1             pyhd3eb1b0_0
xz                        5.2.5                h1a28f6b_1
yaml                      0.2.5                h1a28f6b_0
zlib                      1.2.12               h5a0b063_2

</details>

<details><summary>Logs</summary>

```
# paste relevant logs here, if any
```

File "/Users/eguthman/sleap_m1/sleap/gui/commands.py", line 318, in exportUserLabelsPackage
    self.execute(ExportUserLabelsPackage)
  File "/Users/eguthman/sleap_m1/sleap/gui/commands.py", line 240, in execute
    command().execute(context=self, params=kwargs)
  File "/Users/eguthman/sleap_m1/sleap/gui/commands.py", line 135, in execute
    okay = self.ask(context, params)
  File "/Users/eguthman/sleap_m1/sleap/gui/commands.py", line 1288, in ask
    filename, _ = FileDialog.save(
  File "/Users/eguthman/sleap_m1/sleap/gui/dialogs/filedialog.py", line 60, in save
    filename, filter = QtWidgets.QFileDialog.getSaveFileName(*args, **kwargs)
AttributeError: PySide2.QtWidgets.QFileDialog.getSaveFileName(): unsupported keyword 'filters'

</details>

## Screenshots

<img width="570" alt="Screen Shot 2022-08-10 at 12 56 05 PM" src="https://user-images.githubusercontent.com/32780869/184162606-9955c156-8dc5-493d-a4ca-a9c1018bd4b1.png">

## How to reproduce

1. Go to Predict > Export Labels Package > Labeled frames
2. See error

**Comment by roomrys** [comment]
## Problem Analysis
It seems that we use `filters` as a keyword argument into `QtWidgets.GFileDialog.getSaveFilename`, but the qt documentation states that `filter` is expected. No error is thrown on windows. I believe #866 has a fix for this.

**Comment by roomrys** [comment]
This issue has been resolved in the new release - install SLEAP v1.2.7 [here](https://sleap.ai/installation.html#conda-package).
